module/applications/actor/character-generation-prompt.mjs

```diff
@@ -43,24 +43,46 @@ export default class CharacterGenerationPrompt extends HandlebarsApplicationMixi
     };
   }
 
+  // Define a static method to hold the localization paths
+  static getLocalizationPaths() {
+    return {
+      noNamegiver:         "ED.Dialogs.CharGen.Errors.noNamegiver",
+      noClass:             "ED.Dialogs.CharGen.Errors.noClass",
+      attributes:          "ED.Dialogs.CharGen.Errors.attributes",
+      talentRanksLeft:     "ED.Dialogs.CharGen.Errors.talentRanksLeft",
+      skillRanksLeft:      "ED.Dialogs.CharGen.Errors.skillRanksLeft",
+      notFinished:         "ED.Dialogs.CharGen.Errors.notFinished",
+      maxLanguagesToSpeak: "ED.Dialogs.CharGen.Errors.maxLanguagesToSpeak",
+      maxLanguagesToRead:  "ED.Dialogs.CharGen.Errors.maxLanguagesToRead",
+    };
+  }
+  
   // #region Error Messages
-  static errorMessages = {
-    noNamegiver:      "X.You didn't choose a namegiver. Pretty difficult to be a person then, don't you think?",
-    noClass:          "X.There's no class selected. Don't you wanna be magic?",
-    attributes:       "X. This is just reminder: there are still some unspent attribute points. They will be converted to extra karma.",
-    talentRanksLeft:  "X.There's still some ranks left for your class abilities. Use them, they're free.",
-    skillRanksLeft:   "X.You haven't used all of your skill ranks. Come on, don't be shy.",
-  };
+  static get errorMessages() {
+    const paths = this.getLocalizationPaths();
```

Comment:
Hmm, I can't really see the advantage of using an additional function here. This is a one time use, and it feels like it makes it actually a bit more difficult to read when trying to find localizations? 
Why not just put the localization keys directly in the old structure?

Reply:
it didn't work to add the localization into the old function, this was how i got it to work